include/aikido/planner/kunzretimer/KunzRetimer.hpp

```diff
@@ -54,6 +62,14 @@ class KunzRetimer : public aikido::planner::TrajectoryPostProcessor
       double maxDeviation,
       double timeStep);
```

Comment:
One of these constructors should have the default params as default arguments.

Reply:
Done!